src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java

```diff
@@ -84,22 +84,36 @@ public String uploadFile(@RequestParam("file") MultipartFile file,
         String outputPath = FileStorageService.getOutputPath(filename);
         String nameWithoutExtension = outputFilename.substring(0, outputFilename.lastIndexOf(".") + 1);
         extractMetadata(metadata, outputFilename, fileStorageService);
-        String pathFile = storagePath.substring(0, storagePath.lastIndexOf("\\") + 1);
+        String pathFile = storagePath.substring(0, storagePath.lastIndexOf(System.getProperty("file.separator")) + 1);
+
         if (!(resultTitle.size() > 0)) {
-            insertProjectData(filename, pathFile, checksumLocal, USER_ID);
+            insertAssetData(filename, pathFile, checksumLocal, userID);
         }
         if (metadata.equals("true")) {

```

Comment:
Maybe this is redundant?, metadata should be boolean.

Reply:
yes, but the metadata variable is of type String in this class when the parameter is received